plugins/outputs/influxdb/http_test.go

```diff
@@ -1087,6 +1101,7 @@ func TestDBRPTagsCreateDatabaseCalledOnDatabaseNotFound(t *testing.T) {
 				}
 			},
 			func(w http.ResponseWriter, r *http.Request) {
+				fmt.Println("two:", r.URL.Path, r.FormValue("q"))
```

Comment:
I guess we want to get rid of those debugging left-overs. ;-)

Reply:
removed